src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
 'use strict';
 
 // write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+
+const appendCol = document.querySelector('.append-column');
+const removeCol = document.querySelector('.remove-column');
+
+const limitMax = 10;
+const limitMin = 2;
+
+function checkDisabled(amount, limit) {
+  if (amount === limit) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
```suggestion
  if (amount === limit) {
    return true;
  }
  
  return false;
```
No need for else here. You code will not go further after return anyway